website/docs/setup/installation/manual/from-source.md

```diff
@@ -291,6 +291,62 @@ Building steps:
   this image, and then builds inside the Container. The target binary is located in
   `target/<target triple>/release/vector` like in the previous case.
 
+## Compiling on Windows
+
+The steps to compile Vector on Windows are different from the ones for other operating systems.
+
+1. Install Rust using [`rustup`][urls.rustup]. It would install toolchain for `x86_64-pc-windows-msvc` target on 64-bit machines by default. If you don't have VC++ build tools, the installer would prompt you to install them.
```

Comment:
```suggestion
1. Install Rust using [`rustup`][urls.rustup]. It would install toolchain for `nightly-x86_64-pc-windows-msvc` target on 64-bit machines by default. If you don't have VC++ build tools, the installer would prompt you to install them.
```

Shouldn't this be it?